src/main/webapp/js/instructorCourses.js

```diff
@@ -1,7 +1,31 @@
 $(document).ready(function(){
     $('#ajaxForCourses').trigger('submit');
+    if (typeof moment !== 'undefined') {
+        prepareTimeZoneInput();
+        autoDetectTimeZone();
+    }
 });
 
+/**
+ * Generate time zone <options> using time zone IDs from
+ * Moment-Timezone library.
+ */
+function prepareTimeZoneInput() {
+    var selectElement = document.getElementById(COURSE_TIME_ZONE);
+    moment.tz.names().forEach(function(name) {
+        var o = document.createElement("option");
+        o.text = name;
+        o.value = name;
+        console.log(name);
+        selectElement.appendChild(o);
+    });
+    $("#"+COURSE_TIME_ZONE).selectize({selectOnTab: true});
+}
+
+function autoDetectTimeZone() {
+    var detectedTimeZone = moment.tz.guess();
```

Comment:
This may not be a major issue, but for some reason the timezone that gets detected for my laptop is Asia/Manila. It detects the timezone fine on my handphone though.